readme.md

```diff
@@ -446,6 +446,7 @@
 - [Vehicle Security and Car Hacking](https://github.com/jaredthecoder/awesome-vehicle-security#readme)
 - [Web Security](https://github.com/qazbnm456/awesome-web-security#readme) - Security of web apps & services.
 - [Lockpicking](https://github.com/meitar/awesome-lockpicking#readme) - The art of unlocking a lock by manipulating its components without the key.
+- [Fuzzing](https://github.com/cpuu/awesome-fuzzing#readme) - Fuzzing is widely used by both software security and by quality assurance.
```

Comment:
Ok, so it's widely used, but that's useless info by itself. It doesn't explain **what fuzzing is**.

Reply:
I know what you mean. I changed !
However, there are a lot of items that are not explained enough in the awesome list.
Is this necessary? If not, I will skip it.